### PR TITLE
AzureCP PR S58~S59: Add Plan Parameters for VM Creation

### PR DESCRIPTION
**Pull request by huangpf**
The content to be added to Changelog is as follows:
* Add Plan Parameters for VM Creation

### CODE REVIEW

**Comment by huangpf** [comment]
@amarzavery @yugangw-msft Could you please help review? Thanks.

File: lib/commands/arm/vm/vmProfile._js [hunk 1]
```diff
@@ -132,6 +132,11 @@ __.extend(VMProfile.prototype, {
       vmProfile.diagnosticsProfile = null;
     }
 
+    // Plan
+    if (this.params.plan !== null) {
```
Comment (yugangw-msft) [inline lib/commands/arm/vm/vmProfile._js]: simply to just "this.params.plan"; otherwise you missed the case of "undefined"

**Comment by yugangw-msft** [comment]
a nit; otherwise LGTM

**Comment by huangpf** [comment]
Thanks @yugangw-msft. Addressed.

**Pull request merged**